src/test/java/be/seeseemelk/mockbukkit/inventory/InventoryMockTest.java

```diff
@@ -45,6 +45,41 @@ public void constructor_SetsSize()
 		assertEquals(18, new SimpleInventoryMock(null, 18, InventoryType.CHEST).getSize());
 	}
 
+	@Test
+	public void constructor_SetsSizeInvalid()
+	{
+		int errorCount = 0;
+
+		try
+		{
+			new SimpleInventoryMock(null, -1, InventoryType.CHEST);
+		} catch(AssertionError e) {
+			errorCount++;
+		}
+
+		try
+		{
+			new SimpleInventoryMock(null, 10, InventoryType.CHEST);
+		} catch(AssertionError e) {
+			errorCount++;
+		}
+
+		try
+		{
+			new SimpleInventoryMock(null, 63, InventoryType.CHEST);
+		} catch(AssertionError e) {
+			errorCount++;
+		}
+
+		assertEquals(3, errorCount);
+	}
```

Comment:
ugh, that is really not a good practice to go about things.
Especially because you lose the information about **where** the error happened which is very much not the purpose of a Unit Test.

To test for an exception you would generally make a seperate Test and then use `@Test(expected = SomeException.class)`.